roles/istio/tasks/main.yml

```diff
@@ -1,13 +1,4 @@
 ---
-- name: Create tmp dir
-  tempfile:
-    state: directory
-  register: tmpdir
-  tags:
-    - install
-    - uninstall
-    - upgrade
-
```

Comment:
See that this was removed; however, I don't see where you now register the `tmpdir` variable. Asking as it's still referenced in several other code blocks in this PR.

Reply:
That task was removed because it is redundant. The playbook already creates a global tmpdir here: https://github.com/sassoftware/viya4-deployment/blob/ansible-lint/playbooks/playbook.yaml#L3-L10